docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -181,6 +181,41 @@ to optimize how Determined interacts with PBS:
    define ``CUDA_VISIBLE_DEVICES`` appropriately or provide the ``pbs.slots_per_node`` setting in
    your experiment configuration to indicate how many GPU slots are intended for Determined to use.
 
+-  Configure PBS to report GPU Accelerator type.
+
+   PBS administrators need to set the value for ``resources_available.accel_type`` on each node that
+   has a GPU. Otherwise, the Cluster tab on the Determined Web UI will show the value
+   ``unconfigured`` for the ``Accelerator`` field in the Resource Pool information.
+
+   PBS administrator can use the following set of commands to set the value of
```

Comment:
PBS administrators

Run the formatted text through grammerly.com to catch this kind of thing.

Reply:
Updated and ran the text through Grammarly